src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  @for $starCount from 1 through 5 {
+    &--#{$starCount} > :nth-child(-n + #{$starCount}) {
+      background-image: url(../images/star-active.svg);
+      @extend %stars-size;
+    }
+  }
```

Comment:
You don't need to add these styles for special selectors.
Just add it to `stars__star` class as default.